myhoard/controller.py

```diff
@@ -1248,6 +1265,54 @@ def _mark_failed_restore_backup_as_broken(self) -> None:
 
         self._build_backup_stream(broken_backup).mark_as_broken()
 
+    def _should_schedule_incremental_backup(self) -> bool:
+        incremental_settings = self.backup_settings.get("incremental", {})
+        if not incremental_settings.get("enabled", False):
+            self.log.info("Incremental backup is disabled in configuration")
+            return False
+
+        dow_idx = datetime.datetime.now(datetime.timezone.utc).weekday()
```

Comment:
I guess the `utc` part might cause unexpected behavior if the service is in e.g. Sydney (+11)... Could we maybe try for "fixed interval between full backups that's reset every week"? With a full backup either every Monday or every Sunday?..

Reply:
I see the point, the fact is that all the date time configuration is in UTC already, so backup hour/minute is assumed to be UTC. Thus I think it's OK to assume this is UTC also. If we are to change the way we deal with time, then we will need to change this together.